pipeline_kafka.c

```diff
@@ -986,44 +1008,28 @@ kafka_consume_main(Datum arg)
 				rd_kafka_message_destroy(messages[i]);
 				messages[i] = NULL;
 			}
+
+			/* Flush if we've buffered enough messages or space used by messages has exceeded 8mb */
+			if (messages_buffered >= consumer.batch_size || buf->len > 8388608)
```

Comment:
8MB seems small. We should raise this or else make it configurable. I think the right thing to really do is to incorporate one of these somehow:

* `message.max.bytes` (server side)
* `fetch.message.max.bytes` (consumer configuration property, should be accessible in the `librdkafka` config object)